patch-diff-report-tool/src/main/java/com/github/checkstyle/data/Statistics.java

```diff
@@ -34,14 +34,24 @@
 public class Statistics {
 
     /**
-     * Map storing severity numbers for difference.
+     * Map storing severity numbers for records removed.
      */
-    private Map<String, Integer> severityNumDiff = new HashMap<>();
+    private Map<String, Integer> severityNumRemoved = new HashMap<>();
 
     /**
-     * Map storing module numbers for difference.
+     * Map storing severity numbers for records added.
      */
-    private Map<String, Integer> moduleNumDiff = new HashMap<>();
+    private Map<String, Integer> severityNumAdded = new HashMap<>();
```

Comment:
IMO, we should `Diff` in the name since these are still for the difference line.

Reply:
done